nodeJS/final_project/conclusion.md

```diff
@@ -57,6 +57,6 @@ We want to emphasize again that the learning never really stops but also that yo
 
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-- [Additional frameworks](https://expressjs.com/en/resources/frameworks.html) built on Express for building specific apps (ie. Feathers for chat applications)
-- More on APIs with this [fantastic intro video](https://www.youtube.com/watch?v=oBW_VNg4qD0)
-- Learn More About DevOPs with [this fantastic github repo](https://github.com/MichaelCade/90DaysOfDevOps)
+- Additional [frameworks built on Express](https://expressjs.com/en/resources/frameworks.html) that might be more suited to building certain types of applications.
```

Comment:
Express took down the 3rd party frameworks page a few weeks ago for internal reasons, so the current link is dead. Could you replace it with this new link please?

```suggestion
- Additional [frameworks built on Express](https://web.archive.org/web/20240328030121/https://expressjs.com/en/resources/frameworks.html) that might be more suited to building certain types of applications.
```

Reply:
Hi no problem. Do I click on commit suggestion to submit another pull request? It's my first time doing this.

Reply:
In this case, since I've actually attached a suggestion in the review comment, you can click "commit suggestion" to automatically commit the suggested change.

For future PRs, especially larger ones that might involve multiple files, you'll likely find it easier to do PRs locally, following the workflow from the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md). Then making changes won't be as finicky, as you can just make the changes on your local machine, then commit and push to your fork and they'll automatically update in the PR.
Depends on exactly where in the course you are, since if you're early on in the course then you might not yet be so comfortable with git.

Reply:
Oh yeah I think I could manage to do that. Thank you. I've almost finished foundations.